### PR TITLE
update for django 1.8

### PR DESCRIPTION
**Pull request by eshellman**
sync_transmeta_db needed updates due to changes in django transactions api and backend api

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def read(*rnames):
 
 setup(
     name="django-transmeta",
-    version="0.7.3",
+    version="0.7.3-eshellman",
```
Comment (bartdag) [inline setup.py:41]: How about 0.7.3-python3-django1.8?

**Comment by bartdag** [comment]
Thank you for this pull request: we created this fork to support Python 3, but we never use sync_transmeta_db in our workflow, hence the bugs you had to fix :-) We were surprise someone else was using this fork, but we'll merge your contributions as soon as we agree on the version string!

Thanks again!

**Comment by eshellman** [comment]
transmeta is a dependency of https://github.com/EbookFoundation/fef-questionnaire which is a fork of a fork of https://github.com/eldest-daughter/ed-questionnaire 

I was also not using sync_transmeta_db until I started working on packaging fef-questionnaire and making a demo for it!

your fork seemed to have the most fixes applied. if you like it, why not just bump to 0.7.4

**Comment by eshellman** [comment]
Although, on a closer look it seems it might be better to get rid of sync_transmeta_db and just use makemigrations.

**Comment by bartdag** [comment]
Oooh, that's probably why we never had to use sync_transmeta_db.

Thanks for the explanation about your use case.

I'm hesitant to bump the version unless we change the package name too (which would then be more confusing). We'll go with 0.7.3-python3-0.7.4 and we'll bump the last part if we have to make further releases.

**Pull request merged**